sdks/python/build-requirements.txt

```diff
@@ -14,6 +14,12 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
+
+# TODO: Consider PEP-517/PEP-518 instead of this file (BEAM-8954).
+
+setuptools
+wheel>=0.36.0
```

Comment:
why is this change necessary?

Reply:
these are common build dependencies, however wheel is not installed by default into `venv`, but is installed in virtualenv-created env. without this change, installing from sdist was failing on some python versions.

Reply:
setuptools is still implicitly installed, keeping it explicitly so that we add them when we adopt PEP-517/518 which might require them mentioned explicitly.